EventsExpress/Controllers/UsersController.cs

```diff
@@ -69,7 +67,22 @@ public IActionResult SearchUsers([FromQuery] UsersFilterViewModel filter)
         }
 
         /// <summary>
-        /// This metod have to return UserDto for Admin.
+        /// This method is used to get the number of all users.
+        /// </summary>
+        /// <param name="accountStatus">Param account status is used to determine the number of users with a specific status.</param>
+        /// <returns>The method returns edited gender.</returns>
+        /// <response code="200">Edit is successful.</response>
+        /// <response code="400">Edit process failed.</response>
+        [HttpGet("[action]")]
+        [Authorize(Policy = PolicyNames.AdminPolicyName)]
+        public async Task<ActionResult<int>> Count([FromQuery] int accountStatus)
+        {
+            var count = await _userService.CountUsersAsync((AccountStatus)accountStatus);
```

Comment:
You can avoid casting here if you specify enum tobe of type int. This will bind automatically. You could also do from path here.

Reply:
Done